test/runnable/test22538.c

```diff
@@ -0,0 +1,35 @@
+// https://issues.dlang.org/show_bug.cgi?id=22534
+
+int sun();
+
+int sunlight() { return sun(); }
+
+static int sun() { return 0; }
```

Comment:
This looks backwards to me? A static declaration should never follow a non-static declaration.

Reply:
yeah, that was backwards.